plugins/lv2/sfizz.cpp

```diff
@@ -629,12 +660,14 @@ sfizz_lv2_process_midi_event(sfizz_plugin_t *self, const LV2_Atom_Event *ev)
                             (int)msg[1],
                             msg[2]);
         break;
-    case LV2_MIDI_MSG_CONTROLLER:
+    // Note(jpc) CC must be mapped by host, not handled here.
+    //           See LV2 midi:binding.
+    /*case LV2_MIDI_MSG_CONTROLLER:
```

Comment:
So if the host does not support `midi:binding` this behaves like option 3?

Reply:
That is the case indeed. This would be nicer if the `midi:binding` feature could be checked.

Also and unrelated, this work lacks support of properly saving and restoring CC parameter values, I have to check this next.

Reply:
Is this feature documented anywhere? If not and we're basically doing it live, I think having ways to check for host support is a bit required.

Reply:
> If not and we're basically doing it live

It's true; on the other hand, this benefits LV2 and the community at large, and LV2 sometimes really needs some features like this.

VST3 is, and will remain probably a while, our recommended format of plugin.